test/common/index.js

```diff
@@ -842,6 +842,36 @@ function requireNoPackageJSONAbove(dir = __dirname) {
   }
 }
 
+function spawnPromisified(...args) {
+  let stderr = '';
+  let stdout = '';
+
+  const child = spawnAsync(...args);
+  child.stderr.setEncoding('utf8');
+  child.stderr.on('data', (data) => { stderr += data; });
+  child.stdout.setEncoding('utf8');
+  child.stdout.on('data', (data) => { stdout += data; });
+
+  return new Promise((resolve, reject) => {
+    child.on('close', (code, signal) => {
```

Comment:
consider using `events.once`

Reply:
Does that particularly matter here? Does it affect its eligibility for GC?

Reply:
I mean you can `await once(child, 'close')`,
Just might be more readable, probably won't affect performance